netdata/netdata
# rename DO_NOT_TRACK to DISABLE_TELEMETRY

##### Summary

Сompatibility with DO_NOT_TRACK preserved.

##### Test Plan

- Ensure there are no DO_NOT_TRACK leftovers besides where they are expected

<details>
<summary>check</summary>

```cmd
netdata (env_var_disable_telemetry) $ git grep "[^_]DO_NOT_TRACK"
daemon/anonymous-statistics.sh.in:  [ ! "${DO_NOT_TRACK:-0}" -eq 0 ] ||
daemon/anonymous-statistics.sh.in:  [ -n "$DO_NOT_TRACK" ]; then
netdata-installer.sh:  [ ! "${DO_NOT_TRACK:-0}" -eq 0 ] ||
netdata-installer.sh:  [ -n "$DO_NOT_TRACK" ]; then
packaging/docker/run.sh:  [ ! "${DO_NOT_TRACK:-0}" -eq 0 ] ||
packaging/docker/run.sh:  [ -n "$DO_NOT_TRACK" ]; then
packaging/installer/kickstart.sh:elif [ -n "$DO_NOT_TRACK" ]; then
packaging/installer/kickstart.sh:  NETDATA_DISABLE_TELEMETRY="${DO_NOT_TRACK}"
packaging/makeself/install-or-update.sh:  [ ! "${DO_NOT_TRACK:-0}" -eq 0 ] ||
packaging/makeself/install-or-update.sh:  [ -n "$DO_NOT_TRACK" ]; then
```

</details>

- w/ DO_NOT_TRACK, w/ DISABLE_TELEMETRY and without them
  - docker image
  - static binary
  - install from source (netdata-installer.sh)

##### Additional Information
